### ISSUE TITLE
3D View - Sculpt Mode - Viewport Overlays - Options - Statistics 

### ISSUE THREAD

**Issue by cl3m3c7** [Bforartists/Bforartists#4211]
**Describe the bug**
object stats in sculpt mode not showing the right number.
This is working in blender 4.1.
May be this commit is the culprit? [GPv3: Show stats in edit mode](https://projects.blender.org/blender/blender/commit/2df06a05ae680103c891b991c9336ade98635c03), if so the fix [Fix #120095: Missing stats in sculpt mode](https://projects.blender.org/blender/blender/commit/96ce5a1fd1c1e786e374d9ec3d93e3fc3c3c8c15) needs to be backported to the 4.1 branch

**Desktop (please complete the following information):**
Win10, NVIDIA

**Comment by cl3m3c7** [comment]
yeah, that's the culprit! [contained in 4.1 tag](https://github.com/Bforartists/Bforartists/commit/2df06a05ae680103c891b991c9336ade98635c03)

**Comment by Draise14** [comment]
Will have to test, I have since merged in 4.2 yesterday and if Blender has the fix, we should too.

**Comment by cl3m3c7** [comment]
this is not contained in blender's 4.1 branch.

**Comment by Draise14** [comment]
Typically Blender merges any official branch fixes to the main branch, which is what we merge in. So after the gym and breakfast I'll confirm if it came through.

**Comment by cl3m3c7** [comment]
I am not sure if I am following, but my point is this: the bug is in (blender) main and brush-assets-project branches and so the fix, meaning the fix for this bug will be available in the 4.2 release. In blender this bug is NOT present in 4.1 but it IS present in bforartists's 4.1, so the fix should be backported for the bforartists 4.1.1

**Comment by cl3m3c7** [comment]
👍🏼 and have fun exercising ☺️

**Comment by cl3m3c7** [comment]
yeah it did come!
https://github.com/Bforartists/Bforartists/commit/96ce5a1fd1c1e786e374d9ec3d93e3fc3c3c8c15
closing this I guess!
